diagnostic.md

```diff
@@ -6,45 +6,45 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What are two ways of telling rails where to route a GET request to '/books/4'?
 
 ```rb
-// your response here
+get '/books', to: 'books#4'
 ```
 
 In what file does the above code go?
 
 ```md
-// your response here
+config/routes.rb
 ```
 
 In your own words, what does the command `bundle exec rake db:migrate` do?
 
 ```md
-// your response here
+bundle exec rake db:migrate will take existing information from one table and migrate them into a whole new database.
```

Comment:
`bundle exec rake db:migrate` executes all of the database migration files in `db/migrate` and generates a new database schema file, `db/schema.rb`.